File: deeplynx.models/ResponseDTOs/TagResponseDto.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace deeplynx.models;

public class TagResponseDto
{
    [Column("id")]
    public long Id { get; set; }

    [Column("name")]
    public string Name { get; set; } = null!;

    [Column("project_id")]
    public long? ProjectId { get; set; }

    [Column("last_updated_at", TypeName = "timestamp without time zone")]
    public DateTime LastUpdatedAt { get; set; }

    [Column("last_updated_by")]
    public long? LastUpdatedBy { get; set; }

    [Column("is_archived")]
    public bool IsArchived { get; set; } = false;

    [Column("organization_id")]
    public long OrganizationId { get; set; }
}

File: deeplynx.UI/src/app/(home)/types/responseDTOs/tagResponseDto.ts
export type TagResponseDto = {
  id: number;
  name: string;
  projectId: number;
  lastUpdatedAt?: string | null;
  lastUpdatedBy?: string | null;
  isArchived:boolean;
  createdBy?: string | null;
  createdAt: string; // ISO date string from backend
  archivedAt?: string | null;
};